tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -487,22 +694,38 @@ def get_all_test_cases():
                               CIPHER_SUITE_IANA_VALUE.keys(),
                               NAMED_GROUP_IANA_VALUE.keys(),
                               SIG_ALG_IANA_VALUE.keys()):
-            yield generate_compat_test(client=client, server=server,
+            if(server == 'mbedTLS' or client == 'mbedTLS'):
+                yield generate_compat_test(client=client, server=server,
```

Comment:
```suggestion
            if not any(isinstance(server, MBedtlsBase), isinstance(client, MBedtlsBase)):
                yield generate_compat_test(client=client, server=server,
```

Reply:
Is there benifit if we change it like this?

Reply:
Minor improvement. 

your code looks like C not like python.

Reply:
Not addressed?

Reply:
Actually, I don't like the keywords continue and goto.

Reply:
Update suggestion. let it more easy understand and more like python.